app/services/background.py

```
@@ -517,6 +519,105 @@ async def _balance_sync_service(self):
+
+                                # Broadcast snapshot via WebSocket
+                                try:
+                                    from app.core.websocket import manager
+                                    await manager.broadcast_portfolio_snapshot(
+                                        user_id=str(user.id),
+                                        snapshot_date=snapshot.snapshot_date,
+                                        total_value_usd=float(snapshot.total_value_usd),
+                                        daily_pnl_usd=float(snapshot.daily_pnl_usd),
+                                        daily_pnl_percent=float(snapshot.daily_pnl_percent)
+                                    )
+                                except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)